Resources/Prototypes/Nyanotrasen/Entities/Mobs/Species/felinid.yml

```diff
@@ -123,7 +123,6 @@
       types:
         Blunt: 1
         Slash: 4
-        Piercing: 1
```

Comment:
This has the consequence of reducing total damage.

Reply:
Which was already 1 more damage than any other morphotype (except oni)

Reply:
Yeah, add it to Blunt, I'd say.

Reply:
Given how flat damage reduction works, I'd suggest putting it into slash instead.